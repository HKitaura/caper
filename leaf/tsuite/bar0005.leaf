require leaf;

fun foo( x: int, y: int ): int
{
	0;
}

fun main(): int
{
	foo( 3 );
}
