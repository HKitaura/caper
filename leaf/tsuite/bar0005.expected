leaf: too few argument to function foo; file: tsuite/bar0005.leaf, line: 10, column: 1
